app/views/admin/users/_member_index.html.erb

```diff
@@ -7,34 +7,64 @@
               <header class="flex">
-                <img class="radius-full mr-2" src="<%= user.profile_image_url_for(length: 50) %>" width="40" height="40" alt="" />
-                <div class="flex flex-col">
-                  <h3 class="fs-base"><%= link_to user.name, admin_user_path(user) %></h3>
-                  <span class="color-base-60">@<%= user.username %></span>
+                <a href="<%= admin_user_path(user.id) %>">
+                  <img class="radius-full align-middle" src="<%= user.profile_image_url_for(length: 50) %>" width="40" height="40" alt="<%= user.name %>" />
+                </a>
+                <div class="pl-3 flex-1 fs-s"><%= link_to user.name, admin_user_path(user) %>
+                  <p class="color-base-60">@<%= link_to user.username, admin_user_path(user) %>&nbsp;
+                  </p>
                 </div>
               </header>
               <div class="flex justify-between py-2">
-                <div>standing info</div>
-                <div>role info</div>
+                <div>
+                <% if user.suspended? %>
+                  <span><%= inline_svg_tag("suspended-circle.svg", aria: true, title: "Suspended", class: "mb-1") %>Suspended</span>
+                <% elsif user.warned? %>
+                  <span><%= inline_svg_tag("warned-circle.svg", aria: true, title: "Warned", class: "mb-1") %>Warned</span>
+                <% elsif user.comment_suspended? %>
+                  <span><%= inline_svg_tag("suspended-circle.svg", aria: true, title: "Comment Suspended", class: "mb-1") %>Comment suspended</span>
+                <% elsif user.trusted? %>
+                  <span><%= inline_svg_tag("trusted-circle.svg", aria: true, title: "Trusted", class: "mb-1") %>Trusted</span>
+                <% else %>
+                  <span><%= inline_svg_tag("good-standing-circle.svg", aria: true, title: "Good Standing", class: "mb-1") %>Good standing</span>
+                <% end %>
+                </div>
+                <div>
+                <span class="c-indicator ml-2 crayons-hover-tooltip" data-tooltip="<%= user.current_role %>">
+                  <%= user.current_role %>
+                </span>
+                </div>
               </div>
               <div class="flex justify-between items-end">
                 <dl class="flex gap-2">
                   <div>
-                    <dt class="color-base-60 fw-normal">Last activity</dt>
-                    <dd>placeholder</dd>
+                    <dt class="color-base-60 fw-normal">Latest activity</dt>
+                    <dd><%= user.latest_activity %></dd>
                   </div>
                   <div>
                     <dt class="color-base-60 fw-normal">Joined on</dt>
                     <dd><%= user.registered_at.strftime("%d %b, %Y") %></dd>
                   </div>
                 </dl>
-                <div>Orgs info</div>
+                <div>
+                  <% if user.organization_memberships.any? %>
```

Comment:
I'm not seeing anything in the designs for the Member Index View regarding linking to an organization on organization profile picture click - thoughts?

Reply:
The idea was not to have any interaction there. Just on hover, display names as in designs. Anyway, the idea around displaying Orgz is, they can have an overview of how many orgz members has joined

because in the near future we probably will remove Organizations and add spaces and user groups there. And at that time we can have detailed interaction.

Reply:
Thanks for this, @anujbhavsar96!